kong/db/strategies/postgres/connector.lua

```diff
@@ -461,7 +463,13 @@ end
 function _mt:truncate()
   local i, table_names = 0, {}
 
-  for row in self:iterate(SQL_INFORMATION_SCHEMA_TABLES) do
+  local sql = concat {
+    "SELECT table_name\n",
+    "  FROM information_schema.tables\n",
+    " WHERE table_schema = ", self:escape_literal(self.config.schema), ";"
+  }
```

Comment:
Since we are reusing this statement below, it'd be both less repetitive and more efficient to do:
```lua
-- top-level upvalue
local SQL_INFORMATION_SCHEMA_TABLES = [[
  SELECT table_name FROM information_schema.tables
    WHERE table_schema = %s;
]]

function _mt:truncate()
  local sql = string.format(SQL_INFORMATION_SCHEMA_TABLES,
                            self:escape_literal(self.config.schema))
end 
```

Reply:
It now uses `CURRENT_SCHEMA` in that query, so it is changed to module level static string.